composer.json

```diff
@@ -18,13 +18,13 @@
         "php": ">=7.3",
         "mybuilder/cronos": "~3.0",
         "doctrine/annotations": "1.*",
-        "symfony/console": "~3.0||^4.0||^5.0",
-        "symfony/config": "~3.0||^4.0||^5.0",
-        "symfony/dependency-injection": "~3.0||^4.0||^5.0",
-        "symfony/framework-bundle": "~3.0||^4.0||^5.0",
-        "symfony/http-kernel": "~3.0||^4.0||^5.0",
-        "symfony/property-access": "~3.0||^4.0||^5.0",
-        "symfony/yaml": "~3.0||^4.0||^5.0"
+        "symfony/console": "^3.4 || ^4.0 || ^5.0 || ^6.0",
```

Comment:
Can you remove support for 3.4 please as `getRootNode` will not exist and will throw an exception

Reply:
done